bot/exts/evergreen/snakes/snakes_cog.py

```diff
@@ -945,13 +945,15 @@ async def about_command(self, ctx: Context) -> None:
             title="About the snake cog",
             description=(
                 "The features in this cog were created by members of the community "
-                "during our first ever [code jam event](https://gitlab.com/discord-python/code-jams/code-jam-1). \n\n"
+                "during our first ever "
+                "[code jam event](https://pythondiscord.com/pages/code-jams/code-jam-1-snakes-bot/). \n\n"
                 "The event saw over 50 participants, who competed to write a discord bot cog with a snake theme over "
                 "48 hours. The staff then selected the best features from all the best teams, and made modifications "
                 "to ensure they would all work together before integrating them into the community bot.\n\n"
                 "It was a tight race, but in the end, <@!104749643715387392> and <@!303940835005825024> "
-                "walked away as grand champions. Make sure you check out `!snakes sal`, `!snakes draw` "
-                "and `!snakes hatch` to see what they came up with."
+                f"walked away as grand champions. Make sure you check out `{ctx.prefix}snakes sal` ,"
```

Comment:
```suggestion
                f"walked away as grand champions. Make sure you check out `{ctx.prefix}snakes sal`,"
```
Excessive space sneaked in here.